### PR TITLE
chore(replay): bump sentry to rrweb version 2.19.0

### PR DESCRIPTION
**Pull request by michellewzhang**
introduced changes in https://github.com/getsentry/rrweb/pull/196

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -261,7 +261,9 @@
       "last 3 iOS major versions",
       "Firefox ESR"
     ],
-    "test": ["current node"]
+    "test": [
```
Comment (michellewzhang) [inline package.json:264]: lint :(

File: package.json [hunk 2]
```diff
@@ -106,7 +106,7 @@
     "cronstrue": "^2.50.0",
     "css-loader": "^7.1.1",
     "css-minimizer-webpack-plugin": "^6.0.0",
-    "diff": "5.1.0",
+    "diff": "5.2.0",
```
Comment (billyvg) [inline package.json:109]: why is this changed?

Comment (michellewzhang) [reply package.json:109]: let me merge master

**Review by scttcper — APPROVED** [review]

**Review by billyvg — APPROVED** [review]
